### ISSUE TITLE
update `rpc-handler`

### ISSUE THREAD

**Issue by Keyrxng** [ubiquibot/comment-incentives#39]
This plugin is using my published package and not the official ubiquity package

- `"@keyrxng/rpc-handler": "^1.0.3"` - remove
- `"@ubiquity-dao/rpc-handler": "^1.3.0"` - replace

**Comment by Keyrxng** [comment]
@zugdev can you use the `/start` command again and assign this. I'm unsure why it did not assign you the first time

@gentlementlegen

**Comment by gentlementlegen** [comment]
@Keyrxng Sadly no, you need to live check the logs, we do not retain them. We can consider pushing them to a db but it will be huge. Cloudflare has a log system but it's a paying feature.

**Comment by Keyrxng** [comment]
I see, we can only assume it was a rate limit of some kind considering my comment went through okay. If it happens again I'll make a point of implementing a core plugin logger with SB posting but this is the first time I've seen it

**Comment by 0x4007** [comment]
> @gentlementlegen it's not trivial to access our worker logs is it? i.e need correct access etc

We just have to manually invite their Cloudflare account with read only access I believe.

**Comment by gentlementlegen** [comment]
@0x4007 Yes but we cannot retroactively check the logs, so in this case it would not be helpful.

**Comment by gentlementlegen** [comment]
@zugdev Should have been assigned and this task un-assignable in the first place. There is an issue on that regard [here](https://github.com/ubiquibot/command-start-stop/issues/26). @Keyrxng that would be truly appreciated that you focus on https://github.com/ubiquibot/command-start-stop/pull/19 so this case doesn't happen again.

**Comment by Keyrxng** [comment]
@Dhruv245 @zugdev 

I don't think either PR has correctly implemented the required changes so it's first one to completion gets the reward.

Where the package is used in the codebase, does anything need updated such as init values etc? If you are confident that you have made the required changes and there are no errors introduced request a review from myself please

---

And I will do it later tonight have an apt to get to shortly

**Comment by 0x4007** [comment]
It's strange that the start command is unreliable. Perhaps we can save logs to our database as we used to for review temporarily.

**Comment by 0x4007** [comment]
I just realized that this is the first version of conversation rewards. We should archive this repository as it is deprecated.
